e2e_samples/dataset_versioning/README.md

```diff
@@ -38,6 +38,28 @@ We refer to [LendingClub issued Loans](https://www.kaggle.com/husainsb/lendingcl
         1. Go to provisioned Azure Data Factory, then click [Author & Monitor] button to open development portal.
         1. Click pencil button at left pane, then select [DeltaCopyPipeline].
         1. Cick [Debug] button to run pipeline.
-    1. **[Need update]** Use Databricks to query Delta versions
+    1. Use Databricks to query Delta versions
+        1. Go to Azure Databricks and create a cluster. Remember to tick the "enable credential passthrough" check under the advanced settings if you plan to use the credential passthrough feature. Make sure you have access to Deltalake either with a RBAC role or Access Control List. [detailed steps](./databricks/README.md)
+        1. Create a notebook in Azure Databricks. Select your cluster and execute these scripts below:
+            ```
```

Comment:
do you want to use
```python

instead of simple ``` ?

https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks

Reply:
thanks :) fixed